main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        3) RPM meter
-        4) Instant fuel economy gauge
-    Name 3 things you can do with the:   Instrument Cluster
-        1) Check current speed
-        2) Make sure the passenger seatbelt is fully engaged
-        3) Optimize fuel consumption
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) A/C control
-        2) Fan speed control
-        3) Temperature control
-    Name 3 things you can do with the:   Environment Controls
-        1) Cool the air
-        2) Check inside temperature
-        3) Heat the air
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) Radio
-        2) Music Streaming system
-        3) Navigation system 
-    Name 3 things you can do with the:   Infotainment System
-        1) Listen to satellite radio program
-        2) listen to music streamed from a portable device
-        3) Get directions 
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) protection
-        2) tilt controls
-        3) height control
-    Name 2 things you can do with the:   Seat
-        1) sit on it
-        2) adjust its height
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Orchestral score
+5 properties:
+    1) number of instruments
+    2) displays notes for each instruments
+    3) instrument list
+    4) contains rehearsal marks
+    5) contains tempo markings
+3 things it can do:
+    1) provides roadmap for the conductor
+    2) gives information for the copyist to generate parts
+    3) provides information for hiring musicians
+ */
+
+/*
+Thing 2) Calendar
+5 properties:
+    1) page size
+    2) paper type
+    3) binder type
+    4) contains year information
+    5) displays national holliday dates
```

Comment:
these are actions, not properties that can be represented with a primitive
- [ ] revise